docs/readthedocs/models/input/grid/line.md

```diff
@@ -6,38 +6,114 @@ Representation of an AC line.
-| uuid              | --   |                                                                                                                                                                                        |
-| id                | --   | Human readable identifier                                                                                                                                                              |
-| operator          | --   |                                                                                                                                                                                        |
-| operationTime     | --   | Timely restriction of operation                                                                                                                                                        |
-| nodeA             | --   |                                                                                                                                                                                        |
-| nodeB             | --   |                                                                                                                                                                                        |
-| parallelDevices   | --   | overall amount of parallel lines to automatically <br> construct (e.g. parallelDevices = 2 will build a <br> total of two lines using the specified parameters)                        |
-| type              | --   |                                                                                                                                                                                        |
-| length            | km   |                                                                                                                                                                                        |
-| geoPosition       | --   | Line string of geographical locations describing the <br> position of the line                                                                                                         |
-| olmCharacteristic | --   | Characteristic of possible overhead line monitoring <br> Can be given in the form of `olm:{<List of Pairs>}`. <br> The pairs are wind velocity in x and permissible <br> loading in y. |
+```{eval-rst}
+.. list-table::
+   :widths: 33 33 33
+   :header-rows: 0
+
+
+   * - Attribute
+     - U n i t
+     - Remarks
+
+   * - uuid
+     - –
+     -
+
+   * - id
+     - –
+     - Human readable identifier
+
+   * - operator
+     - –
+     -
+
+   * - operationTime
+     - –
+     - Timely restriction of operation
+
+   * - nodeA
+     - –
+     -
+
+   * - nodeB
+     - –
+     -
+
+   * - parallelDevices
+     - –
+     - overall amount of parallel lines to automatically construct
+       (e.g. parallelDevices = 2 will build a total of two lines
+       using the specified parameters)
```

Comment:
This text creates a large table width. Can this be modified to reduce the table width?

Reply:
Now we have the smaller text again.